docker-dev-setup/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:23.10
+
+WORKDIR /workspace
+
+# Install PHP and other dependencies
+RUN apt update \
```

Comment:
Use `apt-get` instead of `apt` in Dockerfiles as `apt` does not have a stable interface

Reply:
Please also use `apt-get upgrade -y` to upgrade the packages to the latest version